Sources/SceytChatUIKit/Extension/Codable+Extension.swift

```diff
@@ -8,6 +8,15 @@
 
 import Foundation
 
+extension Decodable {
+    init(jsonString: String) throws {
+        guard let data = jsonString.data(using: .utf8) else {
+            throw NSError(domain: "JSONDecoding", code: 0, userInfo: nil)
+        }
+        self = try JSONDecoder().decode(Self.self, from: data)
+    }
+}
+
```

Comment:
I think it is not needed just do as we did an other cases 
Please Check ChatMessage.Attachment.imageDecodedMetadata or ChatChannel.decodedMetadata